pkg/models/iam/am/am.go

```diff
@@ -403,8 +404,9 @@ func (am *amOperator) CreateOrUpdateGlobalRoleBinding(username string, globalRol
 
 	globalRoleBinding := iamv1alpha2.GlobalRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   fmt.Sprintf("%s-%s", username, globalRole),
-			Labels: map[string]string{iamv1alpha2.UserReferenceLabel: username},
+			Name:        fmt.Sprintf("%s-%s", username, globalRole),
+			Labels:      map[string]string{iamv1alpha2.UserReferenceLabel: username},
+			Annotations: map[string]string{constants.KubefedManagedLabel: "false"},
```

Comment:
Federatedresources like  federatedusers is controllered by the user entry in host cluster, modify objects in host cluster and then distribute them through kubefed controller.
```
apiVersion: types.kubefed.io/v1beta1
kind: FederatedUser
metadata:
  finalizers:
  - kubefed.io/sync-controller
  labels:
    kubefed.io/managed: "true"
  name: admin
  ownerReferences:
  - apiVersion: iam.kubesphere.io/v1alpha2
    blockOwnerDeletion: true
    controller: true
    kind: User
    name: admin
```
The user in host cluster needs to set the kubefed managed label to ensure that it will not be affected by the fedkubefed controller